structopt/common/population/predators/__init__.py

```diff
@@ -43,11 +43,11 @@ def post_processing(self):
 
     @staticmethod
     @functools.wraps(best)
-    def best(population, fits, nkeep, **kwargs):
-        return best(population, fits, nkeep, **kwargs)
+    def best(population, fits, nkeep):
+        return best(population, fits, nkeep)
 
     @staticmethod
     @functools.wraps(roulette)
-    def roulette(population, fits, nkeep, **kwargs):
-        return roulette(population, fits, nkeep, **kwargs)
+    def roulette(population, fits, nkeep):
+        return roulette(population, fits, nkeep)
 
```

Comment:
@zhongnanxu I changed this from `**kwargs` back to the actual different keywords for each function. That way they show up correctly in the documentation or via `??` when using ipython.